src/components/policy/policy_external/src/sql_pt_ext_representation.cc

```diff
@@ -879,6 +879,9 @@ bool SQLPTExtRepresentation::GatherApplicationPoliciesSection(
     if (!GatherRequestType(app_id, &*params.RequestType)) {
       return false;
     }
+    if (!GatherRequestSubType(app_id, &*params.RequestSubType)) {
```

Comment:
@a-byzhyn
Don't you think the changes
if (!GatherRequestSubType(app_id, &*params.RequestSubType)) {
return false;
}

should be added also to GatherApplicationPoliciesSection funtion in regular_policy?

Reply:
@AByzhynar 
return false means something wrong with the db.
RequestSubType is an optional parameters in the RPC.
Does the function GatherRequestSubType take that factor into consideration?

Reply:
@EKuliiev `params` is output parameter.